### PR TITLE
[excel] (Requirement Sets) Adding newly released ExcelApiOnline 1.1 APIs

### PR DESCRIPTION
**Pull request by AlexJerabek**
Based on this PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/40830

### CODE REVIEW

**Comment by AlexJerabek** [comment]
Please note that the cross-repo links will not yet work. The ref docs are still waiting on the DefinitelyTyped update.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[49ee5d6](https://github.com/OfficeDev/office-js-docs-pr/commits/49ee5d62d713cd36b604eac4a4701a3634b4f50c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/requirement-sets/excel-api-online-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-OnlineApi/docs/reference/requirement-sets/excel-api-online-requirement-set.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C6%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201912060008261264-1477%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=1Pnch9sNV5kmI9OvEu%2FYiZztyg0DurfD%2Fr6wCNL%2FIpI%3D&st=2019-12-06T00%3A04%3A37Z&se=2020-01-06T00%3A09%3A37Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/requirement-sets/excel-api-online-requirement-set.md [hunk 1]
```diff
@@ -35,7 +35,23 @@ Once the API is in a cross-platform requirement set, you should remove or edit t
...
+The following APIs are currently available for Excel on the web as part of the `ExcelApiOnline 1.1` requirement set.
+
+| Class | Fields | Description |
+|:---|:---|:---|
+|[Comment](/javascript/api/excel/excel.comment)|[mentions](/javascript/api/excel/excel.comment#mentions)|Gets the entities (e.g. people) that are mentioned in comments.|
+||[richContent](/javascript/api/excel/excel.comment#richcontent)|Gets the rich comment content (e.g. mentions in comments). This string is not meant to be displayed to end-users. Your add-in should only use this to parse rich comment content.|
+||[updateMentions(contentWithMentions: Excel.CommentRichContent)](/javascript/api/excel/excel.comment#updatementions-contentwithmentions-)|Updates the comment content with a specially formatted string and a list of mentions.|
+|[CommentMention](/javascript/api/excel/excel.commentmention)|[email](/javascript/api/excel/excel.commentmention#email)|Gets or sets the email address of the entity that is mentioned in comment.|
+||[id](/javascript/api/excel/excel.commentmention#id)|Gets or sets the id of the entity. This is aligned with the id information in `CommentRichContent.richContent`.|
```
Comment (Rick-Kirkham) [inline docs/reference/requirement-sets/excel-api-online-requirement-set.md]: What does "is aligned with" mean in this context? Does this mean "exactly matches" or something else?

Comment (Rick-Kirkham) [inline docs/reference/requirement-sets/excel-api-online-requirement-set.md]: Can we change "the id information" to "the ID information"? (Or has lower case now become standard for this abbreviation?)

File: docs/reference/requirement-sets/excel-api-online-requirement-set.md [hunk 2]
```diff
@@ -35,7 +35,23 @@ Once the API is in a cross-platform requirement set, you should remove or edit t
...
+||[updateMentions(contentWithMentions: Excel.CommentRichContent)](/javascript/api/excel/excel.commentreply#updatementions-contentwithmentions-)|Updates the comment content with a specially formatted string and a list of mentions.|
+|[CommentRichContent](/javascript/api/excel/excel.commentrichcontent)|[mentions](/javascript/api/excel/excel.commentrichcontent#mentions)|An array containing all the entities (e.g. people) mentioned within the comment.|
+||[richContent](/javascript/api/excel/excel.commentrichcontent#richcontent)||
+|[Range](/javascript/api/excel/excel.range)|[moveTo(destinationRange: Range \| string)](/javascript/api/excel/excel.range#moveto-destinationrange-)|Moves cell values, formatting, and formulas from current range to the destination range, replacing the old information in those cells.|
+|[RangeFormat](/javascript/api/excel/excel.rangeformat)|[adjustIndent(amount: number)](/javascript/api/excel/excel.rangeformat#adjustindent-amount-)|Adjusts the indentation of the range formatting. The indent value ranges from 0 to 250.|
```
Comment (Rick-Kirkham) [inline docs/reference/requirement-sets/excel-api-online-requirement-set.md]: Is the indent value measured in characters or pixels?

File: docs/reference/requirement-sets/excel-api-online-requirement-set.md (hunk 1 above)
Comment (AlexJerabek) [reply docs/reference/requirement-sets/excel-api-online-requirement-set.md]: Every place I look in the reference docs uses "id"...

**Comment by AlexJerabek** [comment]
Thanks @Rick-Kirkham. I've adjusted accordingly.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[701c24d](https://github.com/OfficeDev/office-js-docs-pr/commits/701c24dced5262272e6d1683654b975f1bc58f4c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/requirement-sets/excel-api-online-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-OnlineApi/docs/reference/requirement-sets/excel-api-online-requirement-set.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C6%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201912060045125501-1477%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=SYmIvL5gcnpWq1s4UXDzm664PdwNCcwJPqVi%2FS9ENCU%3D&st=2019-12-06T00%3A41%3A26Z&se=2020-01-06T00%3A46%3A26Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[2cd0bb9](https://github.com/OfficeDev/office-js-docs-pr/commits/2cd0bb9cc77e107723b8139325b46f7f559265be)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/requirement-sets/excel-api-online-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-OnlineApi/docs/reference/requirement-sets/excel-api-online-requirement-set.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C6%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201912060052404099-1477%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=kFXDBpDhBFvkyn0t2%2F1dQRF0%2FopGOZmZ7UhL%2Fs9cpvQ%3D&st=2019-12-06T00%3A48%3A48Z&se=2020-01-06T00%3A53%3A48Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**